xclim/indices/_anuclim.py

```diff
@@ -317,7 +319,7 @@ def prcptot_wetdry_quarter(
 
     Returns
     -------
-    xarray.DataArray
+    xarray.DataArray : [mm]
```

Comment:
This will be interpreted by sphinx as name: type

Reply:
Woups, should be a ",".